packages/lambdas/src/ihe-gateway-v2-mock-db-client.ts

```diff
@@ -0,0 +1,30 @@
+import { getEnvVarOrFail } from "@metriport/core/util/env-var";
+import * as Sentry from "@sentry/serverless";
+import { RDSDataClient, ExecuteStatementCommand } from "@aws-sdk/client-rds-data";
+
+const resourceArn = getEnvVarOrFail("DB_RESOURCE_ARN");
+const secretArn = getEnvVarOrFail("DB_SECRET_ARN");
+const region = getEnvVarOrFail("AWS_REGION");
+
+const rdsDataClient = new RDSDataClient({ region });
+
+export const handler = Sentry.AWSLambda.wrapHandler(
+  //eslint-disable-next-line @typescript-eslint/no-explicit-any
+  async (event: any) => {
+    console.log(`Running with event: ${JSON.stringify(event)}`);
+    try {
+      const command = new ExecuteStatementCommand({
+        secretArn,
+        resourceArn,
+        sql: "SELECT * FROM document_query_lambda LIMIT 10",
+        includeResultMetadata: true,
+      });
```

Comment:
following this guide here: 
https://dev.to/slsbytheodo/learn-serverless-on-aws-step-by-step-sql-with-aurora-5hn1